### PR TITLE
Add the ability to switch call audio routes

### PR DESCRIPTION
**Pull request by Naveen3Singh**
### Note:
 - Individual Bluetooth devices are not shown yet (related API methods were added in android pie) and the system decides which Bluetooth device to use.
 - Simple Commons needs to be updated after https://github.com/SimpleMobileTools/Simple-Commons/pull/1549 is merged

### CODE REVIEW

**Comment by tibbi** [comment]
is the current selection somehow highlighted? What I like about radiobuttons is the clearly visible current item.

**Comment by Naveen3Singh** [comment]
nope, but that's a good idea. I'll highlight the selected source with the primary color and maybe also add a tick icon at the end.

**Comment by tibbi** [comment]
it needs some extra indicator to make it totally clear at B&W theme too

**Comment by Naveen3Singh** [comment]
Added a stroke in B&W:

**Comment by tibbi** [comment]
maybe it would make more sense to create a generic equivalent of it in Simple Commons, but do it more like RadioGroup dialog. Thats how those items will mostly be filled out, not by some recyclerview adapter. We will just hardcode the items. Bottom sheet isnt supposed to have too many dynamically generated items that could make it scroll.

**Comment by Naveen3Singh** [comment]
true, the recycler view is there only for nicer (animated) dynamic updates like this (perhaps only valid in the case of dialer, not needed for static items). I guess we'll make it static in commons and dynamic only in the dialer

**Comment by tibbi** [comment]
yep, and make icons optional too, they might not be always used

**Comment by Naveen3Singh** [comment]
yes, icons are optional. Check it now pls

https://github.com/SimpleMobileTools/Simple-Commons/pull/1550 is the related commons PR

File: app/src/main/res/drawable/ic_volume_up_vector.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="24dp" android:height="24dp" android:viewportWidth="24" android:viewportHeight="24">
+    <path android:fillColor="#FFFFFF" android:pathData="M3 10v4c0 0.55 0.45 1 1 1h3l3.29 3.29c0.63 0.63 1.71 0.18 1.71-0.71V6.41c0-0.89-1.08-1.34-1.71-0.71L7 9H4c-0.55 0-1 0.45-1 1zm13.5 2c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-0.73 2.5-2.25 2.5-4.02zM14 4.45v0.2c0 0.38 0.25 0.71 0.6 0.85C17.18 6.53 19 9.06 19 12s-1.82 5.47-4.4 6.5c-0.36 0.14-0.6 0.47-0.6 0.85v0.2c0 0.63 0.63 1.07 1.21 0.85C18.6 19.11 21 15.84 21 12s-2.4-7.11-5.79-8.4C14.63 3.37 14 3.82 14 4.45z"/>
```
Comment (tibbi) [inline app/src/main/res/drawable/ic_volume_up_vector.xml:2]: lets be consistent here, always use @android:color/white or #FFFFFFFF

Comment (Naveen3Singh) [reply app/src/main/res/drawable/ic_volume_up_vector.xml]: Should we also do this in commons? Replacing all usages with "@android:color/white"

Comment (tibbi) [reply app/src/main/res/drawable/ic_volume_up_vector.xml]: I was using #FFFFFFFF mostly, no need to replace it now. Just specify alpha

Comment (Naveen3Singh) [reply app/src/main/res/drawable/ic_volume_up_vector.xml]: Oops, I just did that. Reverting...

Comment (Naveen3Singh) [reply app/src/main/res/drawable/ic_volume_up_vector.xml]: all right :)

**Comment by tibbi** [comment]
will accept it now, but do more tests soon. Thanks

**Comment by tibbi** [comment]
seems to work well :)

**Pull request merged**